### PR TITLE
Fix some tiny browser icons on hidpi screens

### PR DESCRIPTION
**Pull request by nyalldawson**

### CODE REVIEW

**Comment by nyalldawson** [comment]
("some" = "not all")! See http://lists.qt-project.org/pipermail/interest/2018-September/030898.html

File: src/app/qgssettingstree.cpp [hunk 1]
```diff
@@ -60,37 +60,35 @@ QgsSettingsTree::QgsSettingsTree( QWidget *parent )
...
 
-  connect( &refreshTimer, &QTimer::timeout, this, &QgsSettingsTree::maybeRefresh );
+  connect( &mRefreshTimer, &QTimer::timeout, this, &QgsSettingsTree::maybeRefresh );
 }
 
 void QgsSettingsTree::setSettingsObject( QgsSettings *settings )
 {
-  delete this->settings;
-  this->settings = settings;
+  delete this->mSettings;
```
Comment (m-kuhn) [inline src/app/qgssettingstree.cpp]: How is the ownership of `mSettings` managed? It's deleted here, but not in the destructor and the signature also doesn't speak about transferring ownership.

Comment (nyalldawson) [reply src/app/qgssettingstree.cpp]: Note sure. Ping @3nids

Comment (elpaso) [reply src/app/qgssettingstree.cpp]: Generally speaking we shouldn't do this but use `QSettings()...` directly any time we need it, according to Qt docs this is fast enough ( http://doc.qt.io/qt-5/qsettings.html#basic-usage second paragraph )

**Pull request merged**